content/en/docs/guides/auth.md

```diff
@@ -112,6 +112,26 @@ passed to the factory method.
   configured for POSIX filesystems.
 {{% /alert %}}
 
+#### Using OAuth token-based authentication
+
+OAuth 2.0 Protocol is the industry-standard protocol for authorization. It enables
+websites or applications to obtain limited access to user accounts using OAuth tokens.
+
+gRPC offers a set of simple APIs to integrate OAuth 2.0 into applications, streamlining authentication.
+
+At a high level, using OAuth token-based authentication includes 3 steps:
+
+1. Get OAuth token on client side.
+   * You can obtain OAuth tokens using libraries like
+   [oauth2 package](https://pkg.go.dev/golang.org/x/oauth2#section-readme) in Go and
+   [google-auth](https://google-auth.readthedocs.io/en/master/user-guide.html) in Python.
+2. Create Channel credentials or per-call credentials with the OAuth token.
+   * The token will be send to server, normally as part of metadata.
```

Comment:
If you want, you can replace "metadata" with the "HTTP Authorization header".

Reply:
Make sense, I'll use header instead since it's more accurate.